sumologic/resource_sumologic_cloudfront_source_test.go

```diff
@@ -0,0 +1,194 @@
+package sumologic
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+)
+
+func TestAccSumologicCloudFrontSource_create(t *testing.T) {
+	var cloudFrontSource PollingSource
+	var collector Collector
+	cName := acctest.RandomWithPrefix("tf-acc-test")
+	cDescription := acctest.RandomWithPrefix("tf-acc-test")
+	cCategory := acctest.RandomWithPrefix("tf-acc-test")
+	sName := acctest.RandomWithPrefix("tf-acc-test")
+	sDescription := acctest.RandomWithPrefix("tf-acc-test")
```

Comment:
couple questions,
1. what's prefix `c` and `s`?
2. do we need randomized description and category?
3. these are all repeated in every test, can we DRY it?

Reply:
1. `c` - collector, `s` - source
2 & 3: I followed the same pattern as present in other source acceptance tests.